.github/workflows/main.yml

```diff
@@ -0,0 +1,116 @@
+name: Alfresco tomcat base Docker image
+on:
+  push:
+    branches:
+      #TODO add master when confident
+      - opsexp-[0-9]+**
+
+env:
+  IMAGE_REGISTRY_NAMESPACE: alxgomz
+  IMAGE_REPOSITORY: alfresco-base-tomcat
+jobs:
+  docker_images:
+    strategy:
+      matrix:
+        tomcat_major:
+          - 8
+          - 9
+        base_image:
+          - flavor: centos
+            major: 7
+          - flavor: debian
+            major: 11
+          - flavor: ubuntu
+            major: 20.04
+          - flavor: ubi
+            major: 8
+        java_major:
+          - 11
+        jdist:
+          - jre
+        include:
+          - tomcat_major: 8
+            base_image:
+              flavor: centos
+              major: 7
+            java_major: 8
+            jdist: jdk
+          - tomcat_major: 8
+            base_image:
+              flavor: centos
+              major: 7
+            java_major: 11
+            jdist: jdk
+    runs-on: ubuntu-latest
+    steps:
+      - id: co
+        name: checkout project
+        uses: actions/checkout@v2
+      - id: vars
+        name: compute image tag
+        env:
+          IMAGE_BASE_NAME: tomcat${{ matrix.tomcat_major }}-${{ matrix.jdist }}${{ matrix.java_major }}-${{ matrix.base_image.flavor }}${{ matrix.base_image.major }}
+        run: |
+          if [[ "${{ github.ref_name }}" != "master" ]]
+          then
+            echo "::set-output name=image_tag::${IMAGE_BASE_NAME}-${GITHUB_REF_NAME//\//-}"
+          elif [[ "${{ github.event.head_commit.message }}" =~ .*\[release\].* ]]
```

Comment:
not sure if it's a good idea to do an explicit release management for base image, is this working as before?

Reply:
if referring to the "[release] flag in the commit message, yes this is how it's done today